src/ioctls/mod.rs

```diff
@@ -1,12 +1,22 @@
-use kvm_bindings::kvm_run;
-#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-use kvm_bindings::{kvm_cpuid2, kvm_cpuid_entry2};
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: Adding the license is an unrelated change. Can you please add it in a new commit?

Reply:
Done, take a look.